src/components/Header/component.js

```diff
@@ -1,28 +1,29 @@
 import { useContext } from 'react';
+import Logo from './Logo';
 import SearchContext from '../../pages/MovieListPage/SearchContext';
-import logo from '../../assets/images/logo.png';
 import { Outlet } from 'react-router';
```

Comment:
JFYI: `react-router` is core library and using for all types of routers. `react-router-dom` is library for creating router in the web and it is using `react-router` under the hood and installs it automatically as dependency so you do not need to install `react-router` manually or import something directly from `react-router` without specific reasons.